source/includes/_embed2.md.erb

```diff
@@ -388,10 +391,21 @@ Closes the currently-displayed campaign (does nothing if no campaign is currentl
 
 Deletes the `chatter` used to fetch conversation logs for an end-user from storage. When a user opens a new Chat window a new `chatter` will be generated.
 
+### evaluateCampaignConditions
+**This action requires the Ada Engage feature**
+
+`evaluateCampaignConditions(options: CampaignParams): Promise<void>;`
+
+This is similar to `triggerCampaign` - however, instead of triggering a specific Campaign, `evaluateCampaignConditions` evaluates the trigger conditions of the Campaigns in priority order and triggers the first Campaign (if any) whose conditions are matched. This can be useful, for example, if Embed2 cannot determine that the route has changed and the campaign trigger rules need to be evaluated again.
+
+An optional argument `options` can be passed that matches the `CampaignParams` interface. These options may be helpful when testing your Campaign. The options are:
+- `ignoreFrequency?: boolean`: if `ignoreFrequency` is `true`, Campaigns that have already been triggered within the frequency configured on the Campaign will also have their trigger conditions evaluated and may be triggered.
+- `ignoreStatus?: boolean`: if `ignoreStatus` is `true`, inactive and draft Campaigns will also have their trigger conditions evaluated and may be triggered.
+
 ### getInfo
 `getInfo(): Promise<WindowInfo>;`
 
-Returns an object containing information about the bot, including `hasActiveChatter`, `hasClosedChat`, and `isChatOpen`.
+Returns a `WindowInfo` object (see Common Types) containing information about the bot.
```

Comment:
should we link (see Common Types) to the corresponding section?

Reply:
I was thinking about it. You're right, we do it other places, might as well do here.